src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -2506,7 +2506,7 @@ export interface Hacknet {
    * Returns false otherwise.
    *
    * @param index - Index/Identifier of Hacknet Node.
-   * @param n - Number of levels to purchase. Must be positive. Rounded to nearest integer. Defaults to 1 if not specified.
+   * @param n - Number of levels to purchase. Must be positive. Rounded to the nearest integer. Defaults to 1 if not specified.
```

Comment:
![image](https://github.com/bitburner-official/bitburner-src/assets/141260118/de5225ac-0804-44ca-90a1-d53ad0dd1bb4)
Can something be rewritten for clarity?

As an example - given a hacknet-server-1 at Level 1, what outcome should be expected from `ns.hacknet.upgradeLevel(1,20.5)`
Error because 20.5 isn't rounded? upgrade to level 21? to level 22?

Reply:
In general - sure, "nearest integer" refers to a specific, countable noun, making "Rounded to **the** nearest integer" grammatically correct. 

But in the context, it's not the stylistic choice I would make - it's 1 of 4 sentences in a very small space and "nearest integer" is plenty understandable; it's the specific of what is rounded, how and by whom that is unclear.